examples/siro_sandbox/server/interprocess_record.py

```diff
@@ -0,0 +1,55 @@
+import queue
```

Comment:
The HITL server runs a server process that is separate from the main app process. This is important e.g. so that we can receive incoming messages in a timely fashion even when the FPS of the simulation is low. `InterprocessRecord` is meant to encapsulate all this inter-process communication.